lib/slack.rb

```diff
@@ -1,11 +1,158 @@
 #!/usr/bin/env ruby
+require "pry"
+
+require_relative "channel"
+require_relative "user"
+require "httparty"
+require "dotenv"
+Dotenv.load
+
+class SlackError < StandardError; end
+
+class Slack
```

Comment:
This class would be better named `SlackWorkspace` since slack can include multiple workspaces.  

Also it would be good to wrap everything in a module